blocks/image/src/components/media.tsx

```diff
@@ -348,7 +272,7 @@ export const Media: FunctionComponent<
           onCaptionConfirm={() => updateData({ src: draftSrc })}
           onReset={resetComponent}
           width={draftWidth}
```

Comment:
Width is still draggable with `readonly` btw (not sure if it's being persisted)

Reply:
Okay – that's an existing flawing with the block's read only implementation. Will fix.

Reply:
Can be done in follow-up if we like

Reply:
It's fixed already – thanks for flagging